src/components/MainContent.js

```diff
@@ -1,21 +1,39 @@
-import { 
-  BsFillPersonFill, FiMenu, FcSettings, IoNotificationsSharp, FaCouch, BsPersonFill
-} from '../utils/icons';
-import CompanyInfo from './CompanyInfo';
+import {
+  BsFillPersonFill,
+  FiMenu,
+  FcSettings,
+  IoNotificationsSharp,
+  FaCouch,
+  BsPersonFill,
+  ImCheckmark,
+  HiDotsVertical,
+} from "../utils/icons";
+import CompanyInfo from "./CompanyInfo";
+import "./MainContent.css";
 
 export default function MainContent() {
   return (
-    <div className="w-screen h-screen grid grid-cols-2 border-[2px] border-white overflow-y-auto">
-      <div className="w-[calc(100vw-238px)] h-full ml-[218px] grid grid-cols-6 grid-rows-12 border-[2px] border-black">
+    <div className="box w-screen h-screen grid grid-cols-2 overflow-y-auto bg-gray-200">
+      <div className="w-[calc(100vw-238px)] h-full ml-[218px] grid grid-cols-6 auto-rows-min gap-5">
         {/* Nav section */}
-        <div className="col-start-1 col-end-7 row-start-1 row-end-2 flex justify-between border-[2px] border-blue-300 text-sm">
+        <div className="col-start-1 col-end-7 row-start-1 row-end-2 flex justify-between text-sm mb-2 mt-2 p-2">
           <div className="flex flex-col">
             <div>Pages / Dashboard</div>
             <div className="font-semibold">Dashboard</div>
           </div>
-          <div className="flex items-center justify-evenly w-[430px]">
-            <input className="border-[1px] py-[2px] px-[2px] rounded-md border-gray-300" />
-            <button className="border-[1px] rounded-md border-pink-400 text-xs py-[5px] px-[5px] w-[100px] text-pink-400">Online Builder</button>
+          <div className="flex items-center justify-evenly w-[500px]">
+            <label className="w-fit relative">
+              <input
+                className="input border-[1px] w-[175px] h-[35px] py-[2px] px-[8px] rounded-md border-gray-400 bg-transparent focus:outline-none  focus:border-t-transparent placeholder:opacity-0"
```

Comment:
Small nits: remove extra spaces where you can like: `focus:outline-none focus:border-t-transparent`

Reply:
I see see an extra space here. This one wasn't addressed in your commits i think?

Reply:
fixed!